Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -208,59 +209,21 @@ final class MainViewController: BaseViewController {
                 self?.refreshControl.endRefreshing()
             }
         }
-        refreshControl.addAction(action, for: .valueChanged)
-        refreshControl.tintColor = .grey001
-        listCollectionView.refreshControl = refreshControl
+        self.refreshControl.addAction(action, for: .valueChanged)
+        self.refreshControl.tintColor = .grey001
+        self.listCollectionView.refreshControl = self.refreshControl
     }
     
     private func setupSkeletonView() {
-        listCollectionView.showAnimatedGradientSkeleton(usingGradient: .init(colors: [.grey003, .darkGrey002]), animation: skeletonAnimation, transition: .none)
+        self.listCollectionView.showAnimatedGradientSkeleton(usingGradient: .init(colors: [.grey003, .darkGrey002]), animation: skeletonAnimation, transition: .none)
```

Comment:
p2;

```suggestion
        self.listCollectionView.showAnimatedGradientSkeleton(usingGradient: .init(colors: [.grey003, .darkGrey002]),
                                                             animation: skeletonAnimation,
                                                             transition: .none)
```
요로코롬 줄을 바꾸는게 어떨까요 ??

Reply:
[499e6ca](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/375/commits/499e6ca383e5b3a73178117a88f2bb71af859ffd)
반영했습니다.